src/new_app.js

```diff
@@ -0,0 +1,50 @@
+//run with 'node new_web.js', access the site at http://localhost:3000/
+const express = require('express');
+const bodyParser = require('body-parser');
+const OpenAI = require("openai");
+const dotenv = require("dotenv");
+const path = require('path');
+const { add_to_history, retrieve_history } = require('./lib/chat_history');
+
+
+dotenv.config();
+
+const openai = new OpenAI({
+    apiKey: process.env.OPENAI_API_KEY //accessing the api key from the .env file
+});
+const app = express(); //express js for the backend server
+app.use(bodyParser.json())
+; //body-parser extracts the entire body portion of an incoming request stream and exposes it on req.body. https://stackoverflow.com/questions/38306569/what-does-body-parser-do-with-express
+
+
+app.get('/', (req, res) => {
+    res.sendFile(path.join(__dirname, 'new_testing_file.html')); //to serve our html file aka load it when the server starts, can be accessed at http://localhost:3000/ (by default port is set to 3000)
+});
```

Comment:
The new_web.js was the initial name of the new file that I found inappropriate at a certain point and renamed it but I may forgot to rename it in some comments... I'll correct it

Reply:
And the "new" parts of the filenames were meant to be for differentiate from the original ones because I didn't want to just delete the original ones